docs/installation.rst

```diff
@@ -0,0 +1,135 @@
+section.
+The ``dmriprep`` command-line adheres to the `BIDS-Apps recommendations
+for the user interface <usage.html#execution-and-the-bids-format>`__.
+Therefore, the command-line has the following structure:
+::
+
+  $ dmriprep <input_bids_path> <derivatives_path> <analysis_level> <named_options>
+
+On the other hand, if you chose a container infrastructure, then
+the command-line will be composed of a preamble to configure the
+container execution followed by the ``dmriprep`` command-line options
+as if you were running it on a *bare-metal* installation.
+The command-line structure above is then modified as follows:
+::
+
+  $ <container_command_and_options> <container_image> \
+       <input_bids_path> <derivatives_path> <analysis_level> <dmriprep_named_options>
+
+Therefore, once specified, the container options and the image to be run
+the command line is the same as for the *bare-metal* installation but dropping
+the ``dmriprep`` executable name.
+
+Container technologies: Docker and Singularity
+==============================================
+Container technologies are operating-system-level virtualization methods to run Linux systems
+using the host's Linux kernel.
+This is a lightweight approach to virtualization, as compares to virtual machines.
+
+
+.. _installation_docker:
+
+Docker (recommended for PC/laptop and commercial Cloud)
+-------------------------------------------------------
+Probably, the most popular framework to execute containers is Docker.
+If you are to run *dMRIPrep* on your PC/laptop, this is the RECOMMENDED way of execution.
+Please make sure you follow the `Docker installation`_ instructions.
+You can check your `Docker Engine`_ installation running their ``hello-world`` image: ::
+
+    $ docker run --rm hello-world
+
+If you have a functional installation, then you should obtain the following output. ::
+    
+    Hello from Docker!
+    This message shows that your installation appears to be working correctly.
+    
+    To generate this message, Docker took the following steps:
+     1. The Docker client contacted the Docker daemon.
+     1. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+        (amd64)
+     1. The Docker daemon created a new container from that image which runs the
+        executable that produces the output you are currently reading.
+     1. The Docker daemon streamed that output to the Docker client, which sent it
+        to your terminal.
+    
+    To try something more ambitious, you can run an Ubuntu container with:
+     $ docker run -it ubuntu bash
+    
+    Share images, automate workflows, and more with a free Docker ID:
+     https://hub.docker.com/
+    
+    For more examples and ideas, visit:
+     https://docs.docker.com/get-started/
+
+After checking your Docker Engine is capable of running Docker images, then go ahead
+and `check out our documentation <https://fmriprep.readthedocs.io/en/stable/docker.html>`__
```

Comment:
fix link

Reply:
Yup, same as above. I didn't want to copy the whole thing.